src/main/java/se/AppModel/Course.java

```
@@ -0,0 +1,37 @@
+package se.AppModel;
+import java.util.ArrayList;
+public class Course {
+    private String name;
+    private ArrayList<Qualification> requirements;
+    private Teacher assignedTeacher;
+
+    public Course (String n) {
+        name = n; 
+        requirements = new ArrayList<Qualification>();
+    }
+
+    //method to add a teacher to the course (but only if theyre qualifications meet the requirement)
+    // this method iterates through the requirements arraylist and checks if the Teacher has the qualification
+    // if there is a Qualification that is not contained in the teacher's qualifications then the method does nothing 
+    // and returns false 
+    // if the teacher has all of the requirements then they are assigned to the course and the method returns true 
+    // boolean was for testing - most likely will get changed 
+    public boolean assignTeacher(Teacher t) {
```

just a comment, no need to change, but should probably 'throw' something rather than return false, like IllegalArgumentException that would be dealt with in a try/catch block outside the class. I think it's a better OOP principle.